test/e2e/metrics_test.go

```diff
@@ -57,13 +67,29 @@ func TestCatalogdMetricsExportedEndpoint(t *testing.T) {
 		"https://catalogd-service.NAMESPACE.svc.cluster.local:7443/metrics",
 	)
 
-	config.run()
+	mtc.run()
+}
+
+func findK8sClient(t *testing.T) (kubernetes.Interface, *rest.Config) {
+	cfg, err := config.GetConfig()
+	if err != nil {
```

Comment:
nit: maybe just `require.NoError(f)` instead of an `if`?

Reply:
done. and the one below too.